chain/chain/src/chain.rs

```diff
@@ -3451,7 +3451,10 @@ impl<'a> ChainUpdate<'a> {
     where
         F: FnMut(ChallengeBody) -> (),
     {
-        debug!(target: "chain", "Process block {} at {}, approvals: {}, me: {:?}", block.hash(), block.header().height(), block.header().num_approvals(), me);
+        let _span =
+            tracing::debug_span!(target: "chain", "Process block", "#{}", block.header().height())
```

Comment:
@matklad Was there any decision about the use of spans in regards to the fact that we don't see them on segfaults?

Reply:
Not really, easier to just implement the thing than decide: https://github.com/near/nearcore/pull/4916 )